src/utils/jotai/atomWithPartialUpdate.ts

```diff
@@ -0,0 +1,20 @@
+import { PrimitiveAtom, WritableAtom } from 'jotai'
+import { atom } from 'jotai/index'
+
+export function atomWithPartialUpdate<T>(anyAtom: PrimitiveAtom<T>): {
```

Comment:
@alfetopito @nenadV91 @tukantje @anxolin pls, take this into account

Reply:
This is cool!

Reply:
great!